scripts/process_data.py

```diff
@@ -834,13 +383,53 @@ def main(self) -> None:
         CONSOLE.rule()
 
 
+@dataclass
+class StackConfig:
```

Comment:
I'm not a huge fan of the boilerplate here; if we don't want the full traceback, can we just use a try/except in the entrypoint?
```
    try:
        tyro.cli(Commands).main()
    except RuntimeError as e:
        CONSOLE.log("[bold red]" + e.args[0])
```

Reply:
Got it, that makes sense! I have a medium preference for eliminating the extra complexity + layer of inheritance / redirection.

> However I wanted to keep the ability to have the stack trace for debugging purpose and enclosing in a try catch remove that possibility.

My feeling is that it may be good to disentangle errors from incorrect usage (like passing in an invalid argument or directory) vs actual errors (like bugs or dependency issues). To me it makes the most sense to suppress traces for the former case while retaining them for the latter.